src/components/TaskFilters.js

```diff
@@ -231,30 +235,49 @@ export default function TaskFilters() {
     );
 }
 
+function DifficultyFilter({ filterState }) {
+    const selectedDifficulties = filterState.difficulty;
+    const dispatch = useDispatch();
+
+    const toggleDifficulty = difficulty => {
+        const isSelected = selectedDifficulties.includes(difficulty);
+
+        if (isSelected)
+            dispatch(
+                updateTaskFilter({
+                    field: 'difficulty',
+                    value: selectedDifficulties.filter(diffs => diffs !== difficulty),
+                })
+            );
+        else dispatch(updateTaskFilter({ field: 'difficulty', value: [...selectedDifficulties, difficulty] }));
+    };
+
+    return (
+        <div className='grid grid-cols-3 gap-px bg-subdued w-full'>
+            {Object.values(DIFFICULTY).map(({ icon, label }) => (
+                <div
+                    className={`p-1 bg-hover cursor-pointer ${
+                        selectedDifficulties?.includes(label) ? 'bg-secondary text-accent' : 'bg-primary'
+                    }`}
+                    key={label}
+                    onClick={() => toggleDifficulty(label)}
+                >
+                    <img src={icon} alt={label} className='inline mx-1' />
+                    {label}
+                </div>
+            ))}
+        </div>
+    );
+}
+
 function SkillsFilter({ filterState }) {
-    const skillsData = getSkillsPanelData({ exclusions: ['QP', 'Overall', 'Combat'] });
+    const skillsData = [...getSkillsPanelData({ exclusions: ['QP', 'Overall', 'Combat'] }), {}];
```

Comment:
Need the extra empty Object otherwise there's a lovely light grey box in the last space due to how the new layout "borders" are styled 😄